package.json

```diff
@@ -28,12 +28,14 @@
   "devDependencies": {
     "ass": "0.0.4",
```

Comment:
I need to get all of these in line. But to answer, after 0.0.4 some fixes came in, but also added a large set of binary dependencies. jrgm/ass.git#5be99ee undoes that required binary dependency, and keeps the other fixes. So, the coverage numbers are correct in this repo but html display is fugly, since the filenames have full paths and take up half the page.